scripts/generate-changed-recipes

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/python
```

Comment:
```suggestion
#!/usr/bin/env python
```

maybe add `.py` extension to filename?

Reply:
e4b9f7645d70012dfa9f32b5549b840fdbe7fa08